### ISSUE TITLE
Empty installation folders?

### ISSUE THREAD

**Issue by hugot** [bpkg/bpkg#96]
Hi there, first of all, great initiative, I love the idea of having a package manager for bash! I installed bpkg today and decided to try it out on a project. I used `bpkg init` to initialize the project and then decided that it would be nice to try out shpec. I then issued the command  `bpkg install rylnd/shpec`, which resulted in `    info: Install /rylnd/shpec/master from remote https://raw.githubusercontent.com [https://github.com]`. I then decided to take a look in the deps folder:
```
$ ls -R deps
deps:
bin  shpec

deps/bin:

deps/shpec:
package.json
``` 
As you can see the folders are empty except for a package.json  file. Am I doing something wrong here?

Hugo.

**Comment by Potherca** [comment]
This problem seem to be caused by a bug that was introduced when I did a cleanup in #81.

The [`package.json` of `rylnd/shpec`](https://github.com/rylnd/shpec/blob/master/package.json) states that the package should be installed globally. Because of accidental breakage `bpkg` tries to install the package locally, which fails because of another accidental breakage (which also caused #95).

I have opened a MR (#99) to fix these issues.
